preview/Win7Msix/Win7MSIXInstaller/InstallUI.hpp

```diff
@@ -118,3 +141,8 @@ BOOL ChangeText(HWND parentHWnd, std::wstring displayText, std::wstring  message
 //
 // PURPOSE: Increment the progress bar one tick based on preset tick
 void UpdateProgressBar();
+
+// FUNCTION: SendInstallCompleteMsg
+//
+// PURPOSE: Sends the WM_INSTALLCOMPLETE_MSG message to the main window when app installation is complete
+void SendInstallCompleteMsg();
```

Comment:
>SendInstallCompleteMsg [](start = 5, length = 22)

can these be made members of the UI class? I hadn't gotten around to moving all the global stuff into the class, which is why there's so much stuff left as global. I'd rather not add more of this.

Reply:
Moved UI controls and methods to members of UI class. Updated call locations.